src/vs/workbench/api/common/extHostTreeViews.ts

```diff
@@ -467,6 +467,8 @@ export class ExtHostTreeView<T> extends Disposable {
 			tooltip: typeof extensionTreeItem.tooltip === 'string' ? extensionTreeItem.tooltip : undefined,
 			command: extensionTreeItem.command ? this.commands.toInternal(extensionTreeItem.command, disposable) : undefined,
 			contextValue: extensionTreeItem.contextValue,
+			enabled: (<any>element).enabled,
```

Comment:
I think we need a SQL Carbon Edit to identify this - there's one lower in the `payload:` part so maybe we can add one here and then do `// {{SQL CARBON EDIT}} - End` after the  end of this `const item` definition?